lib/commands/build/distribution/install.rb

```
@@ -49,10 +49,32 @@ def call(**options)
                 app_id = response['appId']
 
                 extension = platform == 'ios' ? 'ipa' : 'apk'
-                Logger.info 'Downloading build...'
                 output_name = @options[:output] || "#{@options[:build_id]}.#{extension}"
-                `curl --progress-bar -L '#{download_url}' -o #{output_name} `
-                Logger.info "✅ Build downloaded to #{output_name}"
+
+                if File.exist?(output_name)
+                  Logger.info "Build file already exists at #{output_name}"
+                  print 'Do you want to (i)nstall existing file, (o)verwrite with new download, or (c)ancel? [i/o/c]: '
+                  choice = STDIN.gets.chomp.downcase
+
+                  case choice
+                  when 'i'
+                    Logger.info 'Proceeding with existing file...'
+                  when 'o'
+                    Logger.info 'Downloading new build...'
+                    `curl --progress-bar -L '#{download_url}' -o #{output_name}`
+                    Logger.info "✅ Build downloaded to #{output_name}"
+                  when 'c'
+                    Logger.info 'Operation cancelled'
+                    exit(0)
+                  else
+                    Logger.error 'Invalid choice'
```

nit: you can raise an error and the CLI framework will handle exiting